src/tracks/InstrumentTrack.cpp

```diff
@@ -1444,11 +1444,11 @@ InstrumentTrackWindow::InstrumentTrackWindow( InstrumentTrackView * _itv ) :
 	m_miscView = new InstrumentMiscView( m_track, m_tabWidget );
 
 
-	m_tabWidget->addTab( m_ssView, tr( "ENV/LFO" ), 1 );
-	m_tabWidget->addTab( instrumentFunctions, tr( "FUNC" ), 2 );
-	m_tabWidget->addTab( m_effectView, tr( "FX" ), 3 );
-	m_tabWidget->addTab( m_midiView, tr( "MIDI" ), 4 );
-	m_tabWidget->addTab( m_miscView, tr( "MISC" ), 5 );
+	m_tabWidget->addTab( m_ssView, tr( "ENV/LFO" ), tr( "ENV/LFO" ), "env_lfo_active", "env_lfo_inactive", 1 );
```

Comment:
Agree. LMMS would benefit having more descriptive tooltips.

What about?:

- ENV/LFO => "instrument's envelope and low-frequency oscillator parameters"
- FUNC =>"Chord and arpeggio settings"
- FX =>"Manage instrument's effects"
- MIDI => "Configure instrument's MIDI parameters"
- MISC => "Other instrument parameters"